vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/configuration/ZephyrExporterProperties.java

```diff
@@ -19,17 +19,19 @@
 import java.nio.file.Path;
 import java.util.List;
 
-import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
 import org.vividus.zephyr.model.TestCaseStatus;
 
+@Validated
 @ConfigurationProperties("zephyr.exporter")
 public class ZephyrExporterProperties
 {
     private String jiraInstanceKey;
 
-    @NotBlank(message = "Property 'zephyr.exporter.source-directory' must not be blank")
+    @NotNull(message = "Property 'zephyr.exporter.source-directory' must be set")
```

Comment:
how does this change fix the issue?

Reply:
It helps when property is empty, but in case of a non-existent path, we are facing with NoSuchFileException (just like now)
Property is empty:
![image](https://user-images.githubusercontent.com/18533593/183949137-67618bb0-1fc2-4fbd-9fd6-ab6f251d1fa6.png)

Non-existent path:
![image](https://user-images.githubusercontent.com/18533593/183950291-112bf232-83c0-45af-968b-fcf4f4408189.png)

Reply:
If we want to see clear message, I can add try-catch:

        try
        {
            Files.walkFileTree(zephyrExporterProperties.getSourceDirectory(), crawler);
        }
        catch (IOException e)
        {
            throw new NoSuchFileException("Path to directory with test execution JSON results should be corrected: "
                    + zephyrExporterProperties.getSourceDirectory());
        }
or create new annotation @FileByPathExists